sdk/python/tests/compiler/testdata/basic_no_decorator.yaml

```diff
@@ -58,6 +58,10 @@ spec:
           - -c
         image: python:3.5-jessie
       name: exiting
+      metadata:
+        labels:
+          pipelines.kubeflow.org/kfp_sdk_version: "1.5.0"
```

Comment:
This test will start failing every time a new version is released. Perhaps we can exclude the pod metadata from comparison? It looks like other tests handle that.

Reply:
If we remove the pod metadata from comparison, we can't verify in UT that the KFP sdk really generates those labels.
Based on the name (basic_no_decorator.yaml), I think we only verify metadata info in this UT on purpose.